template/src/components/Designs.js

```diff
@@ -0,0 +1,175 @@
+import React, { Component } from 'react';
```

Comment:
Why is this in the app template code? This seems to be specific to the table library we used in spatial-viewer, and I didn't think we were going to make this a standard table library.